src/Versions.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+    <PropertyGroup>
+        <TargetFrameworkVersion>net7.0</TargetFrameworkVersion>
```

Comment:
TargetFrameworkVersion is a predefined msbuild property. Do NOT use it.
It has been replaced by sdk-style projects using <TargetFramework> or <TargetFrameworks>